.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,24 @@
-<!-- Put a description of what this PR is for here, along with any references to issues that this resolves or contributes to -->
+## Description
 
+_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
+If it fixes a bug or resolves a feature request, be sure to link to that issue._
 
 
-<!-- AUTOCHANGELOG for Downstream PRs.
 
-Please select one of the following "release-note:" headings:
-    - release-note:enhancement
-    - release-note:bug
-    - release-note:feature
-    - release-note:change
-    - release-note:none
+## Type of change
 
-Unless you choose release-note:none, please add a release note.
--->
-**Release Note Template (will be copied)**
+_What type of changes does your code introduce to Appium? Put an `x` in the box that apply._
```

Comment:
```suggestion
_What type of changes does your code introduce to the Prometheus operator? Put an `x` in the box that apply._
```

Reply:
And now everyone knows where I have taken this from :smile: